src/Commands/RunScan.ts

```diff
@@ -152,6 +153,40 @@ export class RunScan implements CommandModule {
         describe:
           'A list of specific headers that should be included into request.'
       })
+      .option('exclude-param', {
+        requiresArg: true,
+        array: true,
+        string: true,
+        describe:
+          'A list of regex patterns for parameter names you would like to ignore during the tests. Example: "Id$"'
+      })
+      .option('exclude-entry-point', {
+        array: true,
+        describe:
+          'A list of JSON strings that contain patterns for entry points you would like to ignore during the tests. ' +
+          'Pass an empty string to remove default exclusions. ' +
+          'To apply patterns for all HTTP methods, you can set an empty array to "methods". ' +
+          'Example: "{ "methods": [], "patterns": "users\\/?$" }"',
```

Comment:
shouldn't patterns be an array? ({ "methods": [], "patterns": ["users\\/?$"] }`)

Reply:
Yeah, according to the contract, it should. There is a typo in the docs